org.eclipse.winery.bpmn4tosca.converter.tobpel/src/test/java/org/eclipse/winery/bpmn2bpel/BPMN4Tosca2BpelTest.java

```diff
@@ -10,13 +10,15 @@
  * Contributors:
  *     Sebastian Wagner - initial API and implementation
  *     ZTE - support of more gateways
- *******************************************************************************/
+ **************************************************************************/
```

Comment:
Why did you remove some stars?

You should add yourself here. Please carefully check http://eclipse.github.io/winery/dev/ToolChain#github---prepare-pull-request again.